test/extended/networking/egressip_helpers.go

```diff
@@ -334,7 +333,7 @@ func createPacketSnifferDaemonSet(oc *exutil.CLI, namespace string, scheduleOnHo
 	f := oc.KubeFramework()
 	clientset := f.ClientSet
 
-	tcpdumpImage, err := util.DetermineImageFromRelease(oc, "network-tools")
+	tcpdumpImage, err := exutil.GetDockerImageReference(oc.ImageClient().ImageV1().ImageStreams("openshift"), "network-tools", "latest")
```

Comment:
@tvardema could you pull this change into a separate PR an submit to origin. The basic idea of the change is to not use the `DetermineImageFromRelease` function as that spins up a pod using the cluster version's image ref. This is not desirable for a multi-arch compute cluster as the cluster version's image ref is a single arch reference (which is a problem which needs fixing by itself). But in any case, this is a simpler way to resolve an image reference rather than spinning up a pod

Reply:
Do you mean just this one change to `egressip_helpers.go`?

Reply:
correct